src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -218,6 +218,12 @@ def put(
 
         result = serializer.validated_data
 
+        if getattr(member.flags, "partnership:restricted"):
+            return Response(
+                {"detail": "This is a partnership user."},
```

Comment:
nit: member not user
Since this can be user facing error can we add more detail to it? Something like: This member is managed by an active partnership and cannot be modified till the end of partnership.

Reply:
sure, makes sense